Resources/config/services.xml

```diff
@@ -36,13 +35,17 @@
             <tag name="kernel.cache_clearer" />
         </service>
 
-        <service id="cmf_seo.cache" alias="cmf_seo.cache.file" />
+        <service id="cmf_seo.cache.file" class="Symfony\Component\Cache\Adapter\FilesystemAdapter" public="false">
+            <argument>cmf_seo</argument>
+        </service>
+
+        <service id="cache.cmf_seo" alias="cmf_seo.cache.file" public="false"/>
```

Comment:
this might look like a strange class name, but this is expected: If someone uses Symfony <3.1, the `cmf_seo.cache.file` service is used for caching. If someone use Symfony 3.1+, they will be able to configure which cache is used with the PSR-6 integration in FrameworkBundle.